scripts/exposure-scanning/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "exposure-scanning",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "type": "module"
```

Comment:
I added a package.json here in so that I could use ES modules, not sure if there is an easier way... Also, unsure if there needs to be a specific value for the `license` attribute here

Reply:
To be clear, here you are publishing an `@bbc/exposure-scanning` package for consumption in Simorgh presumably?

I'd probably use the same licence as the parent psammead package.json: https://github.com/bbc/psammead/blob/latest/package.json#L53

Reply:
Interesting. Perhaps I could take this out of `/scripts` and turn this into a psammead package that could be pulled in to simorgh?